gnssanalysis/_version.py

```diff
@@ -59,9 +66,9 @@ class NotThisMethod(Exception):
 HANDLERS: Dict[str, Dict[str, Callable]] = {}
 
 
-def register_vcs_handler(vcs, method):  # decorator
+def register_vcs_handler(vcs: str, method: str) -> Callable:  # decorator
     """Create decorator to mark a method as the handler of a VCS."""
```

Comment:
One-line sphinx docstring (missing the description of input parameters, what's returned: `:param` `:return`)

Reply:
file autogenerated by the versionner script. so won't be addressed